File: Geek Assistant (Android AIO)/Modules/General/Action/GA_SetTooltipInfo.vb
Module GA_SetTooltipInfo

    Public Sub SetToolTipInfo(ByRef ToolTipName As ToolTip, ByRef obj As Object, ToolTipTitle As String, ToolTipText As String)
        If My.Settings.ShowToolTips Then
            If Not ToolTipTitle = ToolTipName.ToolTipTitle Then ToolTipName.ToolTipTitle = ToolTipTitle
            If Not ToolTipText = ToolTipName.GetToolTip(obj) Then ToolTipName.SetToolTip(obj, ToolTipText)
        Else
            ToolTipName.SetToolTip(obj, Nothing)
        End If
    End Sub
End Module

File: Geek Assistant (Android AIO)/Modules/General/Action/GA_SetTooltipInfo.cs

using System.Windows.Forms;

internal static partial class GA_SetTooltipInfo {
    public static void Run(ref ToolTip ToolTipName, Control control, string ToolTipTitle, string ToolTipText) {
        if (c.S.ShowToolTips) {
            if (ToolTipTitle != ToolTipName.ToolTipTitle)
                ToolTipName.ToolTipTitle = ToolTipTitle;
            if (ToolTipText != ToolTipName.GetToolTip((Control)control))
                ToolTipName.SetToolTip((Control)control, ToolTipText);
        } else ToolTipName.SetToolTip((Control)control, "");
    }
}